packages/components/src/components/ScrollArea/styles/scrollArea.module.scss

```
@@ -13,6 +13,10 @@
     width: 100%;
     height: 100%;
     padding-right: var(--scrollbar-size);
+
+    &[data-show-shadow="true"] {
+        box-shadow: inset 0px -11px 8px 0px var(--box-neutral-color);
```

this one looks a bit odd because it also appears on the right side even if there is no horizontal scroll. idk how we could target the horizontal scroll when it exists, but since usually horizontal scrolling is a bad thing, we could stick to only showing the shadow on the bottom. 

```suggestion
        box-shadow: inset 0 -15px 8px -10px var(--box-neutral-color);
```

Also, the shadow appears even when there is no scroll present... it'd be cool if radix exposed some data attrs to let us know when the content is overflowing. Say we use this in the brand chooser, people with 6 or less brands will end up seeing a weird shadow... maybe we can figure out if there is overflow ourselves and apply this programmatically?

---

let's iterate on this!